ccf_transaction_processor/scripts/utils.py

```diff
@@ -29,13 +28,11 @@ def parse_ledger_url(config = None):
 
     if config:
         (hostname, port) = config["rpc-address"].split(':')
-        host = socket.gethostbyname(hostname)
-        return host, port
+        return hostname, port
 
```

Comment:
why are you assuming hostname? this is still just a host. it might be either an ip address or a dns name. cannot assume either.

Reply:
Right, it should be "host". I'll rename that.